content/docs/user-guide/components/reference/ros2/sensors/common/class-segmentation-configuration.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Class Segmentation Configuration
+linktitle: Class Segmentation Configuration
+description: Robot Operating System (ROS 2) sensor configuration in Open 3D Engine (O3DE).
+---
+
+**Class Segmentation Configuration** is a level entity component that allows to globally configure lidar segmentation classes.
+
+## Provider
+
+[ROS 2 Gem](/docs/user-guide/gems/reference/robotics/ros2)
+
+## Properties
+
+![Class Segmentation Configuration component properties](/images/user-guide/components/reference/robotics/ros2/ros2-class-segmentation-configuration-component.png)
+
+| Property                    | Description                                                                                                                                                                | Values      | Default         |
+|-----------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------|-----------------|
+| **Segmentation classes**    | A list of all segmentation classes. Each class consists of a name, class ID and a class color.                                                                             | List        | Unknown, Ground |
+
+## Usage
+
+Add **Class Segmentation Configuration** to the level  entity; then add the desired segmentation classes. Tag entities associated with each class using their respective class names (See [Tag](/docs/user-guide/components/reference/gameplay/tag.md)).
+Additionally, make sure to enable segmentation in the ROS2 Lidar sensors (See [ROS 2 Lidar Sensor](../ros2-lidar-sensor.md)).
```

Comment:
I would consider adding few images here - it may be a little bit easier to understand, how to setup LiDAR simulation with instance / class segmentation. By this I mean a Tag component matching above Segmentation Configuration component.

Additionally I would slightly extend this text:
- class id and color are available through system component, and used in ROS 2 message from LiDAR
- few words about Unknown and Ground, e.g. entities without Tag will be classified as Unknown
- Ground refers precisely to Terrain, not a mesh being a floor in current project

I would also rephrase it a little bit - it will be more clear if there will be an explicit statement that class name (Segmentation Configuration component) should match the tag (Tag component). This is required to handle class / entity id and color correctly.

Reply:
I've applied most of your suggestions. I did not apply the "class id and color are available through system component, and used in ROS 2 message from LiDAR". Since this page describes the Class Segmentation Configuration component (and its configurable properties) I consider this as self explanatory. The information about published data if added, should be placed on the ROS2 Lidar Sensor page. Since there is no information about published data of other types, I do not see it as necessary (at least within the context of this PR). I also did not rephrase the sentance about the class names as I believe that it already specifies the need of matching names "with their respective segmentation class names" (I have however reworded it a bit so that the whole sentance is a bit more specific).

Reply:
You are correct, I accept this outcome!